### PR TITLE
Remove chai schema

### LINKED ISSUE #2139: Remove chai from end2end tests
This is a follow-up ticket from https://github.com/comit-network/comit-rs/pull/2106

# Problem

After moving to jest we should only use jest matchers and no chai matchers

# Goal

Don't have any chai matchers and remove chai from package.json

# Recommendation

Remove https://github.com/comit-network/comit-rs/blob/a9aa5720fedc297cb7005b9461d727f97911b7ba/api_tests/package.json#L59-L66 (including the types a few lines up) and fix the e2e tests.

Note: straightforward replace, don't refactor everything.

### PR DESCRIPTION
**Pull request by D4nte**
Resolves #2139

### CODE REVIEW

**Comment by thomaseizinger** [comment]
NACK https://github.com/comit-network/comit-rs/pull/2565/commits/915d2c8d969d94269ec27f564860e31c503a2d35.

I don't think we should change to absolute URIs, it doesn't make sense as we will unlikely return an action that points to something outside of cnd. The additional configuration options is just an unnecessary burden.
I would suggest to just patch the JSON schema and maybe raise a PR upstream, even though given the number of unmerged PRs, that is unlikely to be merged.

File: api_tests/tests/rfc003_schema.ts [hunk 1]
```diff
@@ -1,80 +1,24 @@
...
-import {
-    Action,
-    EmbeddedRepresentationSubEntity,
-    Entity,
-    Link,
-} from "comit-sdk";
+import { Action, EmbeddedRepresentationSubEntity, Link } from "comit-sdk";
+import { extendSchemaMatcher } from "../src/schema_matcher";
+
+extendSchemaMatcher();
```
Comment (thomaseizinger) [inline api_tests/tests/rfc003_schema.ts:9]: Is there a more global place where we could do this?

File: api_tests/tests/rfc003_schema.ts [hunk 2]
```diff
@@ -179,59 +118,43 @@ describe("Rfc003 schema swap reject tests", () => {
...
-                {
-                    name: "decline",
-                },
-            ]);
+            expect(bobSwapDetails.properties).toMatchSchema(
+                swapPropertiesJsonSchema
+            );
+            expect(bobSwapDetails.actions.map((action) => action.name)).toEqual(
+                expect.arrayContaining(["accept", "decline"])
+            );
```
Comment (thomaseizinger) [inline api_tests/tests/rfc003_schema.ts:126]: :+1:

File: api_tests/tests/rfc003_schema.ts (hunk 1 above)
Comment (D4nte) [reply api_tests/tests/rfc003_schema.ts:9]: Let's try :)

**Pull request merged**